barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.cpp

```diff
@@ -160,13 +150,12 @@ void ClientIVC::complete_kernel_circuit_logic(ClientCircuit& circuit)
  * @param precomputed_vk
  */
 void ClientIVC::accumulate(ClientCircuit& circuit,
-                           const bool _one_circuit,
+                           [[maybe_unused]] const bool _one_circuit,
```

Comment:
The last remnants of this mechanism will be cleaned out by the aforementioned PR from Cody/Adam